src/software/jetson_nano/display/rotary_encoder/rotary_encoder.py

```diff
@@ -68,11 +68,14 @@ def __init__(
         self.on_counterclockwise_rotate = on_counterclockwise_rotate
         self.on_click = on_click
 
-        self.transitions_per_rotation = len(STATES)
+        self.transitions_per_rotation = len(STATES) - 1
```

Comment:
why this change?

Reply:
I was finding that the fourth state transition was not allows being triggered and this was making the user interface feel very laggy. Adding the -1 has made the user interface feel much more smooth.

Reply:
interesting, please add a quick comment

Reply:
added comment